internal/addressbook/server.go

```diff
@@ -0,0 +1,105 @@
+package addressbook
+
+import (
+	"context"
+	"errors"
+	"flag"
+	apipb "github.com/cyneruxyz/address-book/gen/proto"
+	"github.com/cyneruxyz/address-book/internal/repository"
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"google.golang.org/grpc"
+	"net"
+	"net/http"
+)
+
+const (
+	ErrDataNotFound        = "Data not found"
+	ErrStorageNotResponses = "Storage not response"
+)
+
+var (
+	// comand-line options:
+	// gRPC server endpoint
+	grpcServerEndpoint = flag.String("grpc-server-endpoint", "localhost:9090", "gRPC server endpoint")
```

Comment:
Here and below: please, avoid of hard-coding the values; use constants or variables instead